pandas/tests/indexes/test_category.py

```diff
@@ -581,6 +581,16 @@ def test_is_monotonic(self, data, non_lexsorted_data):
         assert c.is_monotonic_increasing
         assert not c.is_monotonic_decreasing
 
+    @pytest.mark.parametrize('unique, not_unique',
```

Comment:
Might be slightly cleaner to do the parametrize like:
```python
@pytest.mark.parametrize('values, expected', [([1, 2, 3], True), ...])
```
Then just have one `assert` in the test:
```python
assert ci.is_unique is expected
```

Reply:
Yeah, that is cleaner. I've changed it.